### ISSUE TITLE
Redmi note bootlop, stuck on boot logo

### ISSUE THREAD

**Issue by Arsyadi** [BurgerZ/WSM-Project#63]
Hi , would you support a new rom on redmi note? Here for new rom http://en.miui.com/download-218.html#326
My redmi note stuck on boot logo.

**Comment by zdunex25** [comment]
Try lstest xposed app it works well with miui v6.

**Comment by MIUITurkey** [comment]
Please can someone create a folder for Turkish translate .

i tried bu i didnt folder name values-tr

**Comment by zdunex25** [comment]
Create folder yourself, add translation push to git and open pull request.

**Comment by MIUITurkey** [comment]
But i dont know how to create folder

**Comment by zdunex25** [comment]
You did in this commit: https://github.com/BurgerZ/WSM-Project/commit/6401683a5731e11e62ffb8d459976c8858dbc78f I do not know why you removed it.

**Comment by BurgerZ** [comment]
No, commit https://github.com/BurgerZ/WSM-Project/commit/6401683a5731e11e62ffb8d459976c8858dbc78f contains values-tr file, not the folder. 
I think that @MIUITurkey use online editor for editing and creating files (direct from web browser). If so, I recommend using Git for Windows (client for Windows OS) or SourceTree (crossplatform and IMHO the best version control systems client) to sync repos and edit files.

Or just use this method:

![](http://i.stack.imgur.com/n3Wg3.gif)

Just use "values-tr/strings.xml" in the new file creation field. So it will create the folder "values-tr" and the "strings.xml" file inside it.

**Comment by Arsyadi** [comment]
latest xposed also not working, i'm using miui v5 btw.
my device freeze and needs to restart always.
could you add support to redmi note?

**Comment by Arsyadi** [comment]
Working fine with delete all .jex file but my device sometimes freeze and need to be restarted. :(
